metadata-ingestion/src/datahub/ingestion/source/sql/oracle.py

```diff
@@ -81,6 +81,15 @@ def get_sql_alchemy_url(self):
             url = f"{url}/?service_name={self.service_name}"
         return url
 
+    def get_identifier(self: BasicSQLAlchemyConfig, schema: str, table: str) -> str:
+        # schema, table is already normalized, we just to normalize database
+        regular = f"{schema}.{table}"
+        if self.database_alias:
+            return f"{self.database_alias.lower()}.{regular}"
+        if self.database:
```

Comment:
Question - why do we need to lower the database here? I'd prefer to retain casing where possible, and lower at a global place

Reply:
schema, table is already normalized from below
https://github.com/datahub-project/datahub/blob/master/metadata-ingestion/src/datahub/ingestion/source/sql/oracle.py#L100
https://github.com/datahub-project/datahub/blob/master/metadata-ingestion/src/datahub/ingestion/source/sql/oracle.py#L132

so I tried to normalize it inside this function.

but i'think, it's better to normalize at [global place](https://github.com/datahub-project/datahub/blob/master/metadata-ingestion/src/datahub/ingestion/source/sql/sql_common.py#L551) 

i remove lower